nubis/Puppetfile

```diff
@@ -46,14 +46,17 @@ mod 'nubis-confd_site',
     :git => 'https://github.com/gozer/nubis-site_confd.git',
     :ref => '0.1.1'
 
-mod 'srf/fluentd', '0.1.4'
+mod 'srf/fluentd',
```

Comment:
Do these commits have anything to do with the centos support, or just a follow along commit?